install.sh

```diff
@@ -78,151 +121,129 @@ __install() {
+			) &	__load "\t[+] Installing Go"
 		fi
+
 		# Add go to the path
 		export PATH="$PATH:/usr/local/go/bin"
 	else
-		echo -e "\e[31m[-] WARNING: Go has been detected on this system,\e[37m if you
-installed with apt, RITA has only been tested with golang 1.7 which is currently not the
-version in the Ubuntu apt repositories, make sure your golang is up to date
-with 'go version'. Otherwise you can remove with 'sudo apt remove golang' and let this script
-install the correct version for you!
+		echo -e "\e[93m\t[!] WARNING: Go has been detected on this system.\e[0m
+\tIf you installed Go with apt, make sure your Go installation is up
+\tto date with 'go version'. RITA has only been tested with golang
+\t1.7 and 1.8 which are currently not the versions in the Ubuntu
+\tapt repositories. You may remove the old version with
+\t'sudo apt remove golang' and let this script install the correct
+\tversion for you!
 "
-
 		sleep 10s
 	fi
 
-
-	echo -e "[+] Configuring Go dev environment...
-\e[0m"
-
-	sleep 3s
-
 	# Check if the GOPATH isn't set
-	if [ -z "${GOPATH}" ]
+	if [ -z ${GOPATH+x} ]
 	then
-		mkdir -p $HOME/go/{src,pkg,bin}
-		echo 'export GOPATH=$HOME/go' >> $HOME/.bashrc
+		( # Set up the GOPATH
+			mkdir -p $HOME/go/{src,pkg,bin}
+			echo 'export GOPATH=$HOME/go' >> $HOME/.bashrc
+			echo 'export PATH=$PATH:$GOPATH/bin' >> $HOME/.bashrc
+		) & __load "[+] Configuring Go dev environment"
 		export GOPATH=$HOME/go
-		echo 'export PATH=$PATH:$GOPATH/bin' >> $HOME/.bashrc
 		export PATH=$PATH:$GOPATH/bin
-	else
-		echo -e "[-] GOPATH seems to be set, we'll skip this part then for now
-		"
 	fi
 
-	echo -e "[+] Getting the package key and install package for MongoDB...
-"
-
-	sleep 3s
-
-	apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 0C49F3730359A14518585931BC711F9BA15703C6
-
-	echo "deb [ arch=amd64 ] http://repo.mongodb.org/apt/ubuntu $(lsb_release -cs)/mongodb-org/3.4 multiverse" > /etc/apt/sources.list.d/mongodb-org-3.4.list
-
-	apt update -qq
-	apt install -y mongodb-org
-
-	printf "\n[+] Running 'go get github.com/ocmdev/rita...'\n\n"
-
-	# Build RITA
+	# Install MongoDB
+	apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
+	--recv 0C49F3730359A14518585931BC711F9BA15703C6 > /dev/null 2>&1 & \
+	__load "[+] Obtaining the package key for MongoDB"
 
-	apt install -y build-essential
-	go get github.com/ocmdev/rita
-	printf "[+] Installing RITA...\n\n"
-	cd $GOPATH/src/github.com/ocmdev/rita
-	make install
+	echo "deb [ arch=$(dpkg --print-architecture) ] http://repo.mongodb.org/apt/ubuntu $(lsb_release -cs)/mongodb-org/3.4 multiverse" > /etc/apt/sources.list.d/mongodb-org-3.4.list
 
-	printf "[+] Transferring files...\n\n"
-	mkdir $_RITADIR
+	apt-get update > /dev/null & __load "[+] Updating apt"
+	apt-get install -y mongodb-org > /dev/null & __load "[+] Installing MongoDB"
 
-	cp -r etc $_RITADIR/etc
-	cp LICENSE $_RITADIR/LICENSE
+	( # Build RITA
+		go get github.com/ocmdev/rita
```

Comment:
Since this defaults to the `master` branch the install fails down below on line 211 when it expects the new `tables.yaml` file. Is there already a method of using this  script to install a different branch?

Reply:
Unfortunately not while using go get. We would have to manually make the path and git clone to use different branches. Alternatively we could use a proxy? maybe?